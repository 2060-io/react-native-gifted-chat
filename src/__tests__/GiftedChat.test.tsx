import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import { GiftedChat } from '../GiftedChat'

const messages = [
  {
    _id: 1,
    text: 'Hello developer',
    createdAt: new Date(),
    user: {
      _id: 2,
      name: 'React Native',
    },
  },
]

// FIXME: Failing due to React Native Platform.select not properly mocked
it.skip('should render <GiftedChat/> and compare with snapshot', async () => {
  const tree = renderer
    .create(
      <GiftedChat
        messages={messages}
        onSend={() => {}}
        user={{
          _id: 1,
        }}
      />,
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
