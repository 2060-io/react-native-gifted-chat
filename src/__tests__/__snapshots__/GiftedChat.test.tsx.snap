// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <GiftedChat/> and compare with snapshot 1`] = `
<View
  onLayout={[Function]}
  style={
    {
      "flex": 1,
    }
  }
  testID="GC_LOADING_CONTAINER"
/>
`;
